Nodejs/Setup/NodejsTools/NodejsToolsFiles.proj

```diff
@@ -17,6 +17,7 @@
         <File Include="Microsoft.NodejsTools.ProjectWizard.dll" />
         <File Include="Microsoft.NodejsTools.Analysis.dll" />
         <File Include="Microsoft.NodejsTools.TestAdapter.dll" />
+        <File Include="Microsoft.NodejsTools.Telemetry.dll" />
```

Comment:
That will be part of closed repo. None of the closed piece will show in open repo. So if someone builds a setup out of open repo it will end up using DummyTelemetryLogger and won't log anything.